docs/articles/building-source.md

```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS.
+3. Make sure that `Files.Package` is set as the startup item by right-clicking on `Files.Package` in the solution explorer and hitting 'Set as Startup item'.
+4. Run `Files.Package, x64` as `Debug` and it should look like this:
```

Comment:
Should be generic as in "Choose the architecture for your system"

Reply:
How about now?
`Install Microsoft Visual Studio and the UWP Development kit.
Clone the repository and open Files.sln in VS.
Make sure that Files.Package is set as the startup item by right-clicking on Files.Package in the solution explorer and hitting 'Set as Startup item'.
Run Files.Package, x64* as Debug and it should look like this: VS Run config image
You're good to go!
If x64 doesn't work, use the architecture of your system`

Reply:
How about now?
![image](https://user-images.githubusercontent.com/71598437/132376707-bc85ec8b-a265-4b68-aed2-2ad44dad625a.png)